apps/website-25/src/components/homepage/CommunitySection/GovernanceProjects.tsx

```diff
@@ -60,7 +60,7 @@ const GovernanceProjects = () => {
               subtitleClassName="text-base"
               imageSrc={project.imageSrc}
               ctaUrl={project.ctaUrl}
-              className="governance-projects__project"
+              className="governance-projects__project max-w-[323px]"
```

Comment:
I removed this from the Card component itself to allow the cards in the values section to go wider.

This section needs to be changed to use SlideList anyway, so this probably isn't a permanent change. I made an issue for that (#269), which I will probably do as part of the [SlideList refactor](https://github.com/bluedotimpact/bluedot/pull/239)